docs/pages/index.mdx

```diff
@@ -6,102 +6,81 @@ layout: tocless-doc
 videoBanner: fSgCnQrNu78
 ---
 
-Teleport is a certificate authority and access plane for your infrastructure.
-With Teleport you can:
+Teleport is Identity-Native Infrastructure Access for engineers and machines. With Teleport you can:
 
-- Use a single solution to access your SSH servers, Kubernetes clusters,
-  databases, desktops, and web applications. 
-- Define sophisticated access policies for every component of your infrastructure, with fine-grained audit logs and session recordings.
-- Automatically on– and off-board users via integrations with single sign-on
-  providers like GitHub, Okta, and Google Workspace.
+- Replace insecure secrets with true identity.
+- Have a single access solution for SSH, Kubernetes, databases, desktops, web applications, and more.
+- Define RBAC for every component of your infrastructure.
+- Gather audit log events and session recordings.
+- Automatically onboard and offboard users with integrations such as Okta, GitHub, and Google Workspace.
 
 ## Try out Teleport
 
-Quickly see how Teleport works in one of our demo environments.
+- [Teleport Cloud](https://goteleport.com/signup/): Deploy a free cloud trial.
+- [Linux Server](./try-out-teleport/linux-server.mdx): Deploy on a Linux Server.
+- [Docker Compose](./try-out-teleport/docker-compose.mdx): Deploy locally with Docker Compose.
+- [Kubernetes](./try-out-teleport/local-kubernetes.mdx): Deploy locally with Kubernetes.
+- [DigitalOcean](./try-out-teleport/digitalocean.mdx): Deploy with our 1-Click DigitalOcean droplet.
 
-- [Linux Server](./try-out-teleport/linux-server.mdx): Deploy Open Source
-  Teleport on a Linux Server.
-- [Digital Ocean](./try-out-teleport/digitalocean.mdx): Use our 1-Click Digital
-  Ocean droplet to set up Teleport quickly.
-- [Browser Labs](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport using our guided
-  interactive labs.
-- [Docker Compose Lab](./try-out-teleport/docker-compose.mdx): Try Teleport
-  locally using Docker Compose.
-- [Kubernetes Lab](./try-out-teleport/local-kubernetes.mdx): See how Teleport
-  runs on Kubernetes with this local lab.
+Want more details on the [different editions of Teleport](./choose-an-edition/introduction.mdx)?
```

Comment:
Would a new user to the docs know what this means? I think we can be more explicit about where in the process of setting up Teleport a user should choose an edition. Otherwise, this seems like a nonessential step, or less important than trying out Teleport.